zuul-core/src/main/java/com/netflix/netty/common/SourceAddressChannelHandler.java

```diff
@@ -89,53 +79,43 @@
             AttributeKey.newInstance("_local_inet_addr");
 
     /**
-     * The local address of this channel.  This is derived from {@code channel.localAddress()}, or from the
-     * Proxy Protocol preface if provided.  If the address is an IPv6 address, the scope identifier is absent.
-     * Unlike {@link #ATTR_SERVER_LOCAL_ADDRESS}, this value is overwritten with the Proxy Protocol local address
-     * (e.g. the LB's local address), if enabled.
+     * The local address of this channel.  This is derived from {@code channel.localAddress()}, or from the Proxy
+     * Protocol preface if provided.  If the address is an IPv6 address, the scope identifier is absent. Unlike {@link
+     * #ATTR_SERVER_LOCAL_ADDRESS}, this value is overwritten with the Proxy Protocol local address (e.g. the LB's local
+     * address), if enabled.
      */
     public static final AttributeKey<String> ATTR_LOCAL_ADDRESS = AttributeKey.newInstance("_local_address");
 
     /**
-     * The port number of the local socket, or {@code -1} if not appropriate.  Use {@link #ATTR_LOCAL_ADDR} instead.
-     */
-    @Deprecated
-    public static final AttributeKey<Integer> ATTR_LOCAL_PORT = AttributeKey.newInstance("_local_port");
-
-
-    /**
-     * The actual local address of the channel, in string form.  If the address is an IPv6 address, the scope
-     * identifier is absent.  Unlike {@link #ATTR_LOCAL_ADDRESS}, this is not overwritten by the Proxy Protocol message
-     * if present.
+     * The actual local address of the channel, in string form.  If the address is an IPv6 address, the scope identifier
+     * is absent.  Unlike {@link #ATTR_LOCAL_ADDRESS}, this is not overwritten by the Proxy Protocol message if
+     * present.
      *
      * @deprecated Use {@code channel.localAddress()}  instead.
      */
     @Deprecated
-    public static final AttributeKey<String> ATTR_SERVER_LOCAL_ADDRESS = AttributeKey.newInstance("_server_local_address");
+    public static final AttributeKey<String> ATTR_SERVER_LOCAL_ADDRESS = AttributeKey
```

Comment:
Line should be broken at the =

Reply:
Yep, style guide should have picked it. Let me fix.